crxmake.sh

```diff
@@ -59,9 +62,16 @@ sig_len_hex=$(byte_swap $(printf '%08x\n' $(ls -l "$sig" | awk '{print $5}')))
 
 echo "Wrote $crx"
 
+# move crx to artifacts folder in circleci
+if [ $CIRCLE_ARTIFACTS ]; then
```

Comment:
Once the build is finished, we move the files to the artifacts folder, which is the folder that corresponds to the build of the current branch. The folder is accessible in CircleCi (for example: https://circleci.com/gh/rangle/augury/908#artifacts)